### ISSUE TITLE
Helm chart has duplicate securityContext attributes for dapr-sidecar-injector

### ISSUE THREAD

**Issue by slacksach** [dapr/dapr#2784]
## In what area(s)?

 /area test-and-release

## What version of Dapr?

latest

## Expected Behavior

Helm chart should not contain duplicate attributes

## Actual Behavior

Helm chart contains duplicate securityContext definitions

```
        securityContext:
          runAsUser: 1000
        command:
        - "/injector"
        args:
        - "--log-level"
        - info
        - "--metrics-port"
        - "9090"
        securityContext:
          runAsNonRoot: true
```

**Comment by yaron2** [comment]
Thanks for reporting. Would you like to submit a PR for this?

**Comment by slacksach** [comment]
@yaron2 Sure thing, https://github.com/dapr/dapr/pull/2785

**Comment by yaron2** [comment]
Thanks!

**Comment by yaron2** [comment]
Closed via #2785.

Thanks @slacksach!
